jest.setup.js

```diff
@@ -1,3 +1,4 @@
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
This is actually not used in this PR, but I project it being needed for the `<CardElement />` and others, so added it.